modules/form/src/components/adapters/__DEPRECATED__adapters.tsx

```diff
@@ -65,7 +65,7 @@ const ENTER_KEY: string = "Enter";
 export const __DEPRECATED__TextFieldAdapter = (props:FieldRenderProps<any> ): ReactElement => {
 
     const { childFieldProps, input, meta, parentFormProps } = props;
-
+debugger
```

Comment:
Shall we remove the debugger here?

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6418/commits/34b1b04cadee820b1b64ff882b6c969a25b8d4df